README.md

```diff
@@ -1,5 +1,23 @@
 # ffd-info-exchange
 
+## Local installation
+
+This app is designed to run on Python 3.4+. `pyvenv` can manage the dependencies installed with `pip`. With that, you can prepare your development environment by running:
+
+```
+git clone https://github.com/18F/ffd-info-exchange.git
+cd ffd-info-exchange
+pyvenv ffd-env
+source ffd-env/bin/activate
+pip install -r requirements.txt
+createdb ffd-info-exchange
```

Comment:
maybe a note saying that this is a postgres thing and you have to make sure you have that running? wherever it makes sense to include that. could possibly do a pre-install step that covers all the prep needed to be able to run these commands on the first pass but that could get too environmentally specific